recipes/default.rb

```diff
@@ -0,0 +1,71 @@
+#
+# Cookbook:: bluematador_agent
+# Recipe:: default
+#
+# Copyright:: 2018, Blue Matador Inc., All Rights Reserved.
+
+case node[:platform]
+when 'debian', 'ubuntu'
+	apt_update 'update' do
+		action :update
+	end
+	package 'apt-transport-https' do
+		action :install
+	end
+
+	apt_repository 'bluematador-agent' do
+		repo_name 'Blue Matador, Inc.'
+		components ['stable', 'main']
+		distribution nil
+		uri 'https://apt.bluematador.com'
+		keyserver 'hkp://keyserver.ubuntu.com:80'
+		key 'BBED0D5A'
+	end
+when 'centos', 'redhat', 'amazon', 'fedora'
+	yum_repository 'bluematador-agent' do
+	  description 'Blue Matador, Inc.'
+	  baseurl 'http://yum.bluematador.com/'
+	  priority '1'
+	  gpgkey 'http://yum.bluematador.com/OPSBUNKER_RPM_KEY.public'
```

Comment:
https?

Reply:
lower versions of redhat don't allow it, and when I got to thinking about it, we really don't need to care about https, all the binary is public.  In fact it might be nice to just use http for apt too so we can avoid the extra dependency. Thoughts?